config/secure-headers.php

```diff
@@ -379,9 +379,7 @@
 			// Allow image to be directly encoded at the img source parameter
 			'allow' => [
 				'https://maps.wikimedia.org/osm-intl/',
-				'https://a.tile.osm.org/',
-				'https://b.tile.osm.org/',
-				'https://c.tile.osm.org/',
+				'https://tile.osm.org/',
```

Comment:
@qwerty287 shouldn't that be `https://tile.openstreetmap.org/` ? The frontend seems to be using the correct tile URLs though.

Reply:
oups.